docs/rules/no-atomics-waitasync.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "es-x/no-atomics-waitasync"
+description: "disallow the `Atomics.waitAsync` method"
+---
+
+# es-x/no-atomics-waitasync
+> disallow the `Atomics.waitAsync` method
+
+- ❗ <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+- ✅ The following configurations enable this rule: `plugin:es-x/no-new-in-esnext`
+
+## 💡 Examples
+
+⛔ Examples of **incorrect** code for this rule:
+
+<eslint-playground type="bad">
+
+```js
+/*eslint es-x/no-atomics-waitasync: [error, { aggressive: true }] */
+Atomics.waitAsync(i32a, 0, 0, 1000)
+```
+
+</eslint-playground>
+
+## 🔧 Options
+
+This rule has an option.
+
+```yaml
+rules:
+  es-x/no-atomics-waitasync: [error, { aggressive: false }]
```

Comment:
I don't think this rule has an aggressive mode. Could you please remove this from the documentation?

Reply:
[fec1ce0](https://github.com/eslint-community/eslint-plugin-es-x/pull/75/commits/fec1ce009fc7d24dc261dd6bfcd2deb97c19f945) :+1: